barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -141,32 +142,30 @@ template <typename Curve> class IPA {
             const Fr round_challenge = transcript->get_challenge<Fr>("IPA:round_challenge_" + index);
             const Fr round_challenge_inv = round_challenge.invert();
 
-            auto G_lo = GroupElement::batch_mul_with_endomorphism(
-                std::span{ G_vec_local.begin(), G_vec_local.begin() + static_cast<long>(round_size) },
-                round_challenge_inv);
             auto G_hi = GroupElement::batch_mul_with_endomorphism(
```

Comment:
We still consider this G_hi even though it's multiplying by the inverse now?

Reply:
G_hi is referring to multiplication of the higher indices of the vector, not to the power the challenge is used in